src/stream_wrap.h

```diff
@@ -101,6 +111,16 @@ class LibuvStreamWrap : public HandleWrap, public StreamBase {
   static void AfterUvShutdown(uv_shutdown_t* req, int status);
 
   uv_stream_t* const stream_;
+
+#ifdef _WIN32
+  // We don't always an FD that we could look up on the stream_
```

Comment:
Don't always what? :-)

Reply:
This was supposed to read “We don’t always have an FD”. :) Done!